packages/radio/src/radio.css

```diff
@@ -23,7 +23,7 @@ governing permissions and limitations under the License.
 /**
  * Work around for https://github.com/adobe/spectrum-css/issues/2251
  **/
-:host([dir="rtl"]) #button:after {
+:host([dir='rtl']) #button:after {
```

Comment:
I'm not a fan of the lint changes falling into unrelated PRs...

Reply:
agreed! I should be more careful with this 💯